src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -105,6 +105,13 @@ class HuiGenericEntityRow extends LitElement {
                     .datetime=${stateObj.last_changed}
                   ></ha-relative-time>
                 `
+              : this.config.secondary_info === "last-triggered"
```

Comment:
```suggestion
              : this.config.secondary_info === "last-triggered" && stateObj.attributes.last_triggered
```

Reply:
cool, thanks! For the novice in Pr's as I am: should I edit this my PR myself?

Reply:
ok, thanks, done